pkgs/development/python-modules/chromadb/default.nix

```diff
@@ -67,20 +67,20 @@
 
 buildPythonPackage rec {
   pname = "chromadb";
-  version = "1.0.20";
+  version = "1.1.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "chroma-core";
     repo = "chroma";
     tag = version;
-    hash = "sha256-jwgm1IXAyctLzUi9GZfgRMiqAuq1BwpcZp/UMlV2t7g=";
+    hash = "sha256-RVXMjniqZ0zUVhdgcYHFgYV1WrNZzBLW9jdrvV8AnRU=";
   };
 
   cargoDeps = rustPlatform.fetchCargoVendor {
     inherit src;
     name = "${pname}-${version}-vendor";
-    hash = "sha256-A4I0xAGmwF9Th+g8bWEmhCRTAp4Q6GYkejHKDqoczY4=";
+    hash = "sha256-owy+6RttjVDCfsnn7MLuMn9/esHPwb7Z7jXqJ4IHfaE=";
   };
 
   # Can't use fetchFromGitHub as the build expects a zipfile
```

Comment:
`fetchurl` is a bad idea because the hash isn't stable. I suggest using `fetchFromGitHub` and running `zip` in e.g. `postInstall`

Reply:
I'm working on it, but the COVID vaccine has knocked me flat

Reply:
I'm letting the corresponding r-ryantm update go through while I replace `fetchzip`. 

I was able to check out the ui sources from github and zip them, but I'm hitting a permission when the Rust build script tries to copy the zip file into the build directory.

When I'm up and around again, I can step through the build and figure it out.

re: updating `swagger-ui`, 1) the Rust source defaults to using 5.17, 2) I have been able to push it to 5.22, 3) attempts to bump past that have failed to build. Regardless, I will see what it takes to make an upgrade work and/or file a bug upstream.